symengine/tests/basic/test_functions.cpp

```diff
@@ -2304,6 +2309,28 @@ TEST_CASE("Dirichlet Eta: functions", "[functions]")
     r1 = dirichlet_eta(zero);
     r2 = div(one, i2);
     REQUIRE(eq(*r1, *r2));
+
+    r1 = dirichlet_eta(im1);
+    r2 = div(one, i4);
+    REQUIRE(eq(*r1, *r2));
+
+    r1 = dirichlet_eta(i2);
+    r2 = div(pow(pi, i2), integer(12));
+    REQUIRE(eq(*r1, *r2));
+
+    r1 = dirichlet_eta(i4);
+    r2 = div(mul(pow(pi, i4), i7), integer(720));
+    REQUIRE(eq(*r1, *r2));
+
+    r1 = dirichlet_eta(i6);
+    r2 = div(mul(pow(pi, i6), integer(31)), integer(30240));
+    REQUIRE(eq(*r1, *r2));
+
+    r1 = dirichlet_eta(i8);
+    r2 = div(mul(pow(pi, i8), integer(127)), integer(1209600));
```

Comment:
All of the above tests `dirichlet_eta` value, not the derivative. Can you add more derivative tests?